templating.rst

```diff
@@ -384,42 +337,20 @@ to render/extend ``app/Resources/views/base.html.twig``, you'll use the
 Referencing Templates in a Bundle
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-*If* you need to refer to a template that lives in a bundle, Symfony uses a **bundle**:**directory**:**filename**
-string syntax. This allows for several types of templates, each which lives in a
-specific location:
-
-* ``AcmeBlogBundle:Blog:index.html.twig``: This syntax is used to specify a
-  template for a specific page. The three parts of the string, each separated
-  by a colon (``:``), mean the following:
-
-  * ``AcmeBlogBundle``: (*bundle*) the template lives inside the AcmeBlogBundle
-    (e.g. ``src/Acme/BlogBundle``);
-
-  * ``Blog``: (*directory*) indicates that the template lives inside the
-    ``Blog`` subdirectory of ``Resources/views``;
+*If* you need to refer to a template that lives in a bundle, Symfony uses a
+``@bundle/Directory/filename.html.twig``. For example, suppose you have a bundle
```

Comment:
Here looks like a word is missing: `Symfony uses a [...].`

I would have used: `Symfony uses [...].` or `Symfony uses the [...] syntax.`